azure_pipelines/changes.txt

```diff
@@ -3,4 +3,4 @@ Added git config default user. Adding azure pipelines to apps allowed to push
 Allow verbose push
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->